algoliasearch/src/main/java/com/algolia/search/saas/Query.java

```diff
@@ -312,6 +312,20 @@ public Boolean getAllowTyposOnNumericTokens() {
         return parseBoolean(get(KEY_ALLOW_TYPOS_ON_NUMERIC_TOKENS));
     }
 
+    private static final String KEY_CLICK_ANALYTICS = "clickAnalytics";
+
+    /**
+     * @param enabled If set to true, the results will return queryID which is needed for sending click | conversion events. Defaults to false.
+     */
+    public @NonNull
+    Query setClickAnalytics(Boolean enabled) {
```

Comment:
Why not `boolean` (unnecessary boxing)?

Reply:
We've been inconsistent across `Query` on this, worth a refactoring to harmonize

Reply:
Added a task in asana to make it consistent around the client